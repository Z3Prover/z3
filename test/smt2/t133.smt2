
; Copyright (c) 2015 Microsoft Corporation


(set-logic QF_RDL)
(declare-fun x () Real)
(declare-fun y () Int)

(reset)
(set-logic QF_IDL)
(declare-fun x () Int)
(declare-fun y () Real)

