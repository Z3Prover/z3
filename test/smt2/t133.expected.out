(error "line 7 column 21: logic does not support integers")
(error "line 12 column 22: logic does not support reals")
